crates/astria-conductor/tests/blackbox/soft_only.rs

```diff
@@ -0,0 +1,326 @@
+use std::time::Duration;
+
+use astria_conductor::config::CommitLevel;
+use futures::future::{
+    join,
+    join4,
+};
+use tokio::time::timeout;
+
+use crate::{
+    helpers::spawn_conductor,
+    mount_abci_info,
+    mount_executed_block,
+    mount_get_commitment_state,
+    mount_get_filtered_sequencer_block,
+    mount_get_genesis_info,
+    mount_update_commitment_state,
+};
+
+#[tokio::test]
+async fn simple() {
+    let test_conductor = spawn_conductor(CommitLevel::SoftOnly).await;
+
+    mount_get_genesis_info!(
```

Comment:
Some of the macros can probably be combined. For example, mounting the genesis info and (initial) commitment state could be made neater by using one macro. Mounting block execution + updating the commitment state also make sense as one macro.